cms/envs/common.py

```diff
@@ -2930,6 +2941,18 @@ def _should_send_xblock_events(settings):
         'learning-certificate-lifecycle':
             {'event_key_field': 'certificate.course.course_key', 'enabled': False},
     },
+    "org.openedx.learning.course.passing.status.updated.v1": {
+        "learning-badges-lifecycle": {
+            "event_key_field": "course_passing_status.course.course_key",
+            "enabled": _should_send_learning_badge_events,
+        },
+    },
+    "org.openedx.learning.ccx.course.passing.status.updated.v1": {
+        "learning-badges-lifecycle": {
+            "event_key_field": "course_passing_status.course.course_key",
```

Comment:
Shouldn't this be master_course_key or ccx_course_key?

Reply:
You're correct, I will update it to ccx_course_key according to event spec

Reply:
I have updated value in lms and cms settings, as checks are passed it should be good to be merged!

Reply:
Thank you!